index.bs

```diff
@@ -4177,15 +4446,16 @@ To <dfn>generate a report URL</dfn> given a [=suitable origin=] |reportingOrigin
 To <dfn>generate an attribution report URL</dfn> given an [=attribution report=] |report| and an optional
 [=boolean=] <dfn for="generate an attribution report URL"><var>isDebugReport</var></dfn> (default false):
 
+1. [=Assert=]: |report| is an [=event-level report=] or an [=aggregatable report=].
```

Comment:
What else could it be?

Reply:
Supposed to be aggregatable attribution report, updated.